components/schema-migrator/migrations/director/202012081537_ord-service-initial.up.sql

```diff
@@ -0,0 +1,302 @@
+             NULL::uuid     AS api_definition_id,
+             NULL::uuid     AS event_definition_id,
+             elements.value AS value
+      FROM packages,
+           jsonb_array_elements_text(packages.tags) AS elements) AS package_tags
+UNION ALL
+(SELECT NULL::uuid         AS package_id,
+        api_definitions.id AS api_definition_id,
+        NULL::uuid         AS event_definition_id,
+        elements.value     AS value
+ FROM api_definitions,
+      jsonb_array_elements_text(api_definitions.tags) AS elements)
+UNION ALL
+(SELECT NULL::uuid     AS package_id,
+        NULL::uuid     AS api_definition_id,
+        id             AS event_definition_id,
+        elements.value AS value
+ FROM event_api_definitions,
+      jsonb_array_elements_text(event_api_definitions.tags) AS elements);
+
+CREATE VIEW countries AS
+SELECT *
+FROM (SELECT packages.id    AS package_id,
+             NULL::uuid     AS api_definition_id,
+             NULL::uuid     AS event_definition_id,
+             elements.value AS value
+      FROM packages,
+           jsonb_array_elements_text(packages.countries) AS elements) AS package_countries
+UNION ALL
+(SELECT NULL::uuid         AS package_id,
+        api_definitions.id AS api_definition_id,
+        NULL::uuid         AS event_definition_id,
+        elements.value     AS value
+ FROM api_definitions,
+      jsonb_array_elements_text(api_definitions.countries) AS elements)
+UNION ALL
+(SELECT NULL::uuid     AS package_id,
+        NULL::uuid     AS api_definition_id,
+        id             AS event_definition_id,
+        elements.value AS value
+ FROM event_api_definitions,
+      jsonb_array_elements_text(event_api_definitions.countries) AS elements);
+
+CREATE VIEW package_links AS
+SELECT id                   AS package_id,
+       actions.type         AS type,
+       actions."customType" AS custom_type,
+       actions.url          AS url
+FROM packages,
+     jsonb_to_recordset(packages.package_links) AS actions(type TEXT, "customType" TEXT, url TEXT);
+
+CREATE VIEW api_resource_definitions AS
+SELECT *
+FROM (SELECT id                                  AS api_definition_id,
+             api_res_defs.type                   AS type,
+             api_res_defs."customType"           AS custom_type,
+             format('/api/%s/specification', id) AS url,
+             api_res_defs."mediaType"            AS media_type
+      FROM api_definitions,
+           jsonb_to_recordset(api_definitions.api_definitions) AS api_res_defs(type TEXT, "customType" TEXT,
+                                                                               "mediaType" TEXT,
+                                                                               url TEXT)) as api_defs
+UNION ALL
+(SELECT id                                  AS api_definition_id,
+        CASE
+            WHEN spec_type::text = 'ODATA' THEN 'edmx'
+            WHEN spec_type::text = 'OPEN_API' THEN 'openapi-v3'
```

Comment:
in ORD there are Open API v2 and v3 but Compass supports only a single `OPEN_API` type - it's not trivial to derive reasoning about what might be the actual version, but using v3 should be good enough for legacy applications resources.